packages/common/integration/tests/pipes/entity-name-pipe.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed } from '@angular/core/testing';
+import { NameByPathPipe, NameByPathPipeModule } from '@angular-ru/common/pipes';
+
+describe('entity name pipe', () => {
+    let pipe: NameByPathPipe;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({ imports: [NameByPathPipeModule] }).compileComponents();
+        pipe = TestBed.inject(NameByPathPipe);
+    });
+
+    it('should return entity name', () => {
+        expect(pipe.transform('a')).toEqual('a');
+        expect(pipe.transform([])).toEqual('');
+        expect(pipe.transform(null)).toEqual('');
+        expect(pipe.transform({ name: 'a' }, 'name')).toEqual('a');
+        expect(pipe.transform({ value: { name: 'a' } }, 'value.name')).toEqual('a');
+        expect(function () {
```

Comment:
Why not to use an arrow function? Something like this: `expect(() => ...).toThrow(...)`

Reply:
I will replace it with an arrow function